make_release/nu_release.nu

```diff
@@ -1,69 +1,60 @@
 use std log
 
-def publish [
-    crate: path # the path to the crate to publish.
-    --no-verify # don’t verify the contents by building them. Can be useful for crates with a `build.rs`.
-] {
-    cd $crate
-
-    if $no_verify {
-        cargo publish --no-verify
-    } else {
-        cargo publish
+export def main [] {
+    let subcrates_wave_1 = [
+        nu-glob,
+        nu-json,
+        nu-path,
+        nu-pretty-hex,
+        nu-system,
+        nu-utils,
+        nu-term-grid,
+        nu-test-support,
+        nu-protocol,
+        nu-engine,
+        nu-plugin,
+        nu-color-config,
+        nu-parser,
+        nu-std,
+        nu-table,
+        nu-cmd-base,
+    ]
+
+    # This crate has a `build.rs` file and thus needs `--no-verify`
+    let subcrates_wave_2 = [
+        nu-cmd-lang,
+    ]
+
+    let subcrates_wave_3 = [
+        nu-command,
+        nu-explore,
+        nu-cli,
+        nu-cmd-dataframe,
+        nu-cmd-extra,
+        nu-lsp,
+
+        nu_plugin_query,
+        nu_plugin_inc,
+        nu_plugin_gstat,
+        nu_plugin_formats,
+    ]
+
+    log warning "starting publish"
+
+    log warning "publishing the first wave of crates"
+    for crate in $subcrates_wave_1 {
+        cargo publish -p $crate
```

Comment:
Ah that's nicer than accidentally ending up in the wrong directory on failure which we encountered in the past. (shouldn't be a problem with the current script due to `def` scoping, and is nicer anyway.